File: backend/components/metiers/threed/legacy/samples/monitoring/monitoring_sample.py
# monitoring_sample.py – Exemple de monitoring ultra avancé


def monitoring_sample(metric, value):
    """Simule une alerte monitoring legacy Threed"""
    # ... logique de monitoring avancée ...
    return (
        f"ALERT: {metric} dépasse le seuil!"
        if value > 100
        else f"OK: {metric}"
    )

File: backend/components/metiers/threed/legacy/samples/monitoring/monitoring_sample.js
// monitoring_sample.js – Exemple de monitoring ultra avancé
/**
 * Simule une alerte monitoring legacy Threed
 */
function monitoringSample(metric, value) {
  // ... logique de monitoring avancée ...
  return value > 100 ? `ALERT: ${metric} dépasse le seuil!` : `OK: ${metric}`;
}

module.exports = monitoringSample;
